specification/ai/Azure.AI.Unified/credentials/models.tsp

```diff
@@ -0,0 +1,56 @@
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@typespec/openapi";
+
+using TypeSpec.OpenAPI;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Azure.AI.Unified;
+
+@doc("The different Credential types")
+union CredentialType {
+  string,
+  ApiKey: "ApiKey",
+  AAD: "AAD",
+  SAS: "SAS",
+}
+
+@doc("Base Credential definition")
+@discriminator("type")
+model BaseCredential {
```

Comment:
Any reason this is not defined in the \connections\models.tsp? I don't see this being used anywhere but for connections